spec/linter-eslint-spec.js

```diff
@@ -84,23 +84,23 @@ async function getNotification(expectedMessage) {
 }
 
 async function makeFixes(textEditor) {
-  return new Promise(async (resolve) => {
+  const editorReloaded = new Promise((resolve) => {
     // Subscribe to the file reload event
-    const editorReloadSub = textEditor.getBuffer().onDidReload(async () => {
-      editorReloadSub.dispose()
-      // File has been reloaded in Atom, notification checking will happen
-      // async either way, but should already be finished at this point
+    const reloadSubscription = textEditor.getBuffer().onDidReload(() => {
```

Comment:
I'd prefer the old variable name since it's more descriptive of what it actually is.

Reply:
I changed it because I felt like `Sub' was a meaningless abbreviation until I dug into the code to work it out. But I can also see why you would want it to specify what is reloading. It's a little on the long side, but how about `editorReloadSubscription`?

Reply:
`editorReloadSubscription` works for me.